src/envo/envo-edit.owl

```diff
@@ -16314,18 +16316,28 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00004683> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00005738> (foam)
 
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_00005738> "https://en.wikipedia.org/wiki/Foam"^^xsd:string)
```

Comment:
`foam` and `sea foam` are showing up here - will have to keep an eye on this when merging

Reply:
Yes, this is a problem - I'd revert changes on this PR or simply close it and start a new one @meichen-liu @raissameyer